lib/endpoints/ChampionEndpoint.js

```diff
@@ -0,0 +1,39 @@
+const Endpoint = require ('./Endpoint');
+const EndpointUtil = require('./EndpointUtil');
+
+/***
+ * Endpoint to receive client information about champions on the league platform
+ * For game data about champions use static-data instead
+ */
+class ChampionEndpoint extends Endpoint {
+
+	constructor(config, rateLimiter){
+		super('Champion', config, rateLimiter);
+		this.apiUrl += `/platform/${this.apiVersion}/champions`;
+	}
+
+	/**
+	 * Gets basic platform information about all champions.
+	 * @param freeToPlay
+	 * @param platformId
+	 * @return {Bluebird<ChampionListDto>}
+	 */
+	gettingList( {freeToPlay = false} = {}, platformId = this.config.PLATFORM_ID){
```

Comment:
Im naming them this way since they are all asynchronous returning a callback.
gettingX implies that it returns a Promise whereas getX implies that it is synchronous or takes a callback.